blackfootwords/datatables.py

```diff
@@ -1,9 +1,24 @@
 from sqlalchemy.orm import joinedload
 from clld.web import datatables
 from clld.web.datatables.base import LinkCol, Col, LinkToMapCol
+from clld.web.datatables.value import Values
+from clld.db.util import get_distinct_values
 
 from blackfootwords import models
 
 
+class Lemmas(Values):
```

Comment:
To customize a datatable, we create a derived class, inheriting from the default implementation.